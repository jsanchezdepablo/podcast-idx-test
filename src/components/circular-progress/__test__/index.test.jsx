import React from "react";
import { shallow } from "enzyme";
import DefaultCircularProgress from "..";

describe("DefaultCircularProgress Component", () => {
  it("Matches snapshot", () => {
    const wrapper = shallow(<DefaultCircularProgress />);
    expect(wrapper).toMatchSnapshot();
  });

  it("test", () => {
    const wrapper = shallow(<DefaultCircularProgress />);
    expect(wrapper.find(".circular-progress__container").exists()).toBeTruthy();
  });
});
