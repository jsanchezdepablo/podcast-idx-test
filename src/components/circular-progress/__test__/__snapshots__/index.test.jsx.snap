// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefaultCircularProgress Component Matches snapshot 1`] = `
<div
  className="circular-progress__container"
>
  <ForwardRef(CircularProgress) />
</div>
`;
